noise_raster/raster_processing.py

```diff
@@ -446,6 +474,29 @@ def set_nodata_value(in_ds):
     # Set no data value for source rasters to -99.0
     dst_ds.GetRasterBand(1).SetNoDataValue(newndv)
     dst_ds.GetRasterBand(1).WriteArray(band1)
+
+    return in_ds
+
     dst_ds = None
+    in_ds = None
+
+def reproject_3035(in_ras, out_ras):
+    """
+    Reproject the final output raster to EPSG:3035.
+    """
+
+    # Add reprojected tif name and file extension to directory file path
```

Comment:
I take this as an example for a general remark on source code comments:

Generally, code comments should preferrably explain _why_ something is done / what the intention is. It should not explain _what_ is happening if the code is self-explainable in that regard (like in this case, [see also](https://en.wikipedia.org/wiki/Comment_(computer_programming)#Code_description)).